src/utils/mockWalkApi.ts

```
@@ -0,0 +1,271 @@
+];
+
+/**
+ * Mock API 응답 헬퍼 함수
+ */
+const createSuccessResponse = <T>(data: T, message: string): ApiResponse<T> => ({
+  httpStatus: 200,
+  message,
+  data
+});
+
+const createErrorResponse = (status: number, message: string): ApiErrorResponse => ({
```

ESLint

[error] 'createErrorResponse' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)